services/core-web/src/styles/settings/themeOverride.scss

```diff
@@ -637,3 +637,8 @@ select {
 .ant-descriptions-item-label {
   font-weight: 600;
 }
+
+.ant-tooltip-disabled-compatible-wrapper {
+  width: inherit;
```

Comment:
Was this used in the end? If not, remove

Reply:
removed